job-server/src/spark.jobserver/io/JobFileDAO.scala

```diff
@@ -135,7 +135,7 @@ class JobFileDAO(config: Config) extends JobDAO {
     new File(rootDir, createJarName(appName, uploadTime) + ".jar").getAbsolutePath
 
   private def createJarName(appName: String, uploadTime: DateTime): String =
-    appName + "-" + uploadTime.toString().replace(':', '_')
+     appName + "_SNAPPY_JOB_SERVER_JAR_" + uploadTime.toString().replace(':', '_')
```

Comment:
Since we will need to register these with executors in most case (SnappyExecutor), will it not be better to register these with the name provided here to install-jar and then use replace-jar for subsequent calls? The additional handling of URLs here and in snappydata looks dicey to me.